backend/internal/consumer/storage_updater.go

```diff
@@ -58,13 +59,13 @@ readLoop:
 			var untypedRecord UntypedRecord
 			err := json.Unmarshal(msg, &untypedRecord)
 			if err != nil {
-				log.Printf("Failed to unmarshal: %v", err)
+				zap.L().Info("Failed to unmarshal: ", zap.Error(err))
```

Comment:
This is a failure, but it doesn't lead to program crashing. Change the logging level to `Error`.